rest/route/github.go

```diff
@@ -314,6 +314,7 @@ func (gh *githubHookApi) AddIntentForPR(pr *github.PullRequest, owner, calledBy
 	if err != nil {
 		return errors.Wrap(err, "failed to create intent")
 	}
+	//change something
```

Comment:
why?

Reply:
Just in case evergreen had some internal logic for ignoring empty PRs that i didn't know about haha